Content.Server/GameTicking/GameTicker.Lobby.cs

```diff
@@ -45,7 +45,8 @@ private string GetInfoText()
             var mapName = map?.MapName ?? Loc.GetString("game-ticker-no-map-selected");
             var gmTitle = Loc.GetString(Preset.ModeTitle);
             var desc = Loc.GetString(Preset.Description);
-            return Loc.GetString("game-ticker-get-info-text",("roundId", RoundId), ("playerCount", playerCount),("mapName", mapName),("gmTitle", gmTitle),("desc", desc));
+            var roundTime = _gameTiming.RealTime.ToString("hh\\:mm");
+            return Loc.GetString("game-ticker-get-info-text",("roundId", RoundId), ("playerCount", playerCount),("mapName", mapName), ("roundTime", roundTime), ("gmTitle", gmTitle),("desc", desc));
```

Comment:
Is RealTime reset on round restart? I didn't think content touched any of the engine times.

Reply:
Looks like it doesn't, I'll just change it to RoundDuration